applications/nrf5340_audio/src/audio/sw_codec_select.h

```diff
@@ -46,13 +46,13 @@ enum sw_codec_select_ch {
 struct sw_codec_encoder {
 	bool enabled;
 	int bitrate;
-	enum sw_codec_select_ch channel_mode;
+	uint8_t num_ch;
 	enum audio_channel audio_ch; /* Only used if channel mode is mono */
 };
 
 struct sw_codec_decoder {
 	bool enabled;
-	enum sw_codec_select_ch channel_mode;
+	uint8_t num_ch;
```

Comment:
```suggestion
	enum sw_codec_num_ch;
```

Reply:
I don't want to pose this limitation as the number of channels can be N.